src/DiskIO/DiskDaemon/DiskdAction.cc

```diff
@@ -65,13 +65,13 @@ DiskdAction::Create(const Mgr::CommandPointer &aCmd)
 DiskdAction::DiskdAction(const Mgr::CommandPointer &aCmd):
     Action(aCmd), data()
 {
-    debugs(79, 5, HERE);
+    debugs(79, 5, "");
```

Comment:
> I see much harm from doing that because it would be impractical to reproduce those manual changes across many branches.

The reduction comes from a) the branch not having a conflict with the automated changes, and b) a regular branch update/rebase reducing the changes whether manual or automated on master repo down to an automated branch change. For branches there is no difference between an automated change upstream or a manual one, the conflicts will be the same and the proposed script cannot be used mid-way during a rebase conflict resolution or it will corrupt the branch changeset being updated.

Branches cannot simply run the script now and be done because that would *add* conflicts on every script changed line that had some difference between what the script does and what the upstream code currently looks like. They have to hold off running it until after merging the specific upstream revision that had the script changes included - and have already faced all conflicts that change causes. All the proposed script does for branches is automate away the requirement that newly added lines not use HERE.

By combining the HERE change with the other polishing into one commit the branches then only face a conflict once IF they change that line themselves. Doing them separately will result in N conflicts for each affected line.